jamjar/jamjar/common/video_utils.py

```diff
@@ -0,0 +1,56 @@
+from django.conf import settings
+
+import uuid
+import os
+
+import logging
+
+from jamjar.tasks.transcode_video import transcode_video
+
+class VideoUtils(object):
+    BASE_PATH = settings.VIDEOS_PATH
```

Comment:
We could actually probably move a lot of these utils into the Video model.  Things that shouldn't be in the Video model can stay here but should be static methods.